src/connector/src/connector_common/common.rs

```diff
@@ -744,3 +744,24 @@ pub(crate) fn load_private_key(
         .ok_or_else(|| anyhow!("No private key found"))?;
     Ok(cert?.into())
 }
+
+#[serde_as]
+#[derive(Deserialize, Debug, Clone, WithOptions)]
+pub struct MongodbCommon {
```

Comment:
This folder is designed for use with the same configuration as sink and source, and it seems to only be used by sink, so it can be moved to the sink directory

Reply:
In there future, we may implement a native version of MongoDB-CDC by leveraging the shared source design and this is the reason why I put some of the common configurations here :)